packages/core/package.json

```diff
@@ -24,12 +24,14 @@
     },
     "dependencies": {
         "async-retry": "1.3.3",
-        "chalk": "2.3.1"
+        "chalk": "2.3.1",
+        "glob": "11.0.0"
```

Comment:
FYI node now supports OOTB glob (but it requires specific node version)

I'm not asking to change it, but just sharing knowledge hehehe

Reply:
Oh that's super nice, I didn't know.
I'm planning on bumping node version at some point this quarter.

Reply:
Edit: it's still experimental TIL – [link](https://nodejs.org/api/fs.html#fsglobpattern-options-callback)

<img width="514" alt="image" src="https://github.com/user-attachments/assets/8d37a217-7a54-4bd4-976d-81a3281e683a" />